### PR TITLE
Styles Review

### PR DESCRIPTION
**Pull request by Felipeg005**
To achieve the requirements next actions were taken:

1. We use styles following the general layout.
2. We concentrate on the general layout, rather than the details.
3. We use plain CSS and a CSS framework (Bootstrap).

Thanks for your review.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#books {
+  max-height: 60vh;
+  max-width: 60vw;
+  overflow-y: scroll;
```
Comment (Lameck1) [inline styles.css]: - [x] The design of this section is kind of not in line with the provided wireframe. Look at this screenshot from the project requirements list. 

| The Requirement |
| --- |
| theirs |

| The Implementation |
| --- |
| yours |

Please consider re-writing styles for this section to make them match the wireframe.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @Felipeg005 & @muhenge , 

So far so good. You people are doing a splendid job 👍🏼. Let us just address one small issue and we'll be good to go.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Felipeg005) [reply styles.css]: Of course, you are totally right, Now we fixed the styles to make them match with the wireframe :+1: 

Thank you :smile:

**Review by hemantso — APPROVED** [review]
Hi Team, 

Good work on making the requested changes. 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**